stream-platform/src/main/java/io/camunda/zeebe/stream/api/ReadonlyStreamProcessorContext.java

```diff
@@ -19,4 +19,6 @@ public interface ReadonlyStreamProcessorContext {
    * @return partition ID
    */
   int getPartitionId();
+
+  boolean enableAsyncScheduledTasks();
```

Comment:
❌ Since this is part of the stream-platform API (boundary with engine), we must add some JavaDoc.